frontend/tests/playwright/product-list/collections_scroll.spec.ts

```diff
@@ -1,14 +1,15 @@
-import { test, expect } from '@playwright/test';
+import { test, expect } from '../test-fixtures';
 // Assert that when a user clicks the pagination buttons,
 // the page scrolls back up.
 
 test.describe('collections scroll', () => {
-   test.beforeEach(async ({ page }) => {
+   test.beforeEach(async ({ page, productPage }) => {
```

Comment:
I think `productPage` in most of the collection tests is unused.

Reply:
Addressed in [`2245ca5` (#1258)](https://github.com/iFixit/react-commerce/pull/1258/commits/2245ca5bbdae207f0e9ea7bb5da0dd4abf376c2f)